app/src/main/java/io/homeassistant/companion/android/settings/sensor/SensorSettingsViewModel.kt

```diff
@@ -94,5 +86,14 @@ class SensorSettingsViewModel @Inject constructor(
                     .mapNotNull { sensor -> sensorsList.firstOrNull { it.id == sensor.id } }
             }
             .associateBy { it.id }
+
+        allSensors = managers.associateWith { manager ->
+            availableSensors = manager.getAvailableSensors(app)
```

Comment:
I think you can immediately return this, no need to put it in a variable?

Reply:
We access the data from the variable once its ready though?

Reply:
oh sorry I see what you mean now :)

Reply:
The variable is local / only available in this function, assigned here and then immediately returned if I understand correctly. So what does the variable add vs. returning the assigned value directly?

Reply:
You are correct, my bad I misinterpreted which line the comment was about 😅